test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/condition/OpenShiftProjectDoesNotExists.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2007-2019 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v 1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.reddeer.condition;
+
+import org.eclipse.reddeer.common.condition.AbstractWaitCondition;
+import org.jboss.tools.openshift.core.connection.Connection;
+import org.jboss.tools.openshift.reddeer.utils.DatastoreOS3;
+import org.jboss.tools.openshift.reddeer.view.OpenShiftExplorerView;
+import org.jboss.tools.openshift.reddeer.view.resources.OpenShift3Connection;
+
+/**
+ * 
+ * @author jkopriva@redhat.com
+ * 
+ */
+public class OpenShiftProjectDoesNotExists extends AbstractWaitCondition {
```

Comment:
Grammar typo. ...DoesNotExist :) But would it not be better to change the wait condition to positive meaning? also shorter name. Then you can use waitWhile instead or waitUntuil and vice versa.

Reply:
Grammar fixed, there is a condition OpenShiftProjectExists so I would leave it as it is.